recipes/pyside2ads/meta.yaml

```diff
@@ -0,0 +1,63 @@
+{% set version = "3.8.2.1" %}
+package:
+  name: pyside2ads
+  version: {{ version }}
+
+source:
+  url: https://github.com/metgem/PySide2Ads/archive/v{{ version }}.tar.gz
+  sha256: 39b75119212dbc465590f11b5e36909949fcc34006d96b843205a4971e8c4bde
+  patches:
+    - 0001-libcxx.patch              # [linux]
+
+build:
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv
+  script_env:
+    - CMAKE_GENERATOR_PLATFORM=      # [win]
+    - CMAKE_GENERATOR_TOOLSET=       # [win]
+    - CMAKE_GENERATOR=               # [win]
+
+requirements:
+  build:
+    - cmake
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - clangdev
```

Comment:
My guess is that clangdev is only needed on macOS and it will be installed by the previous `compiler` requirement.
Shouldn't that be removed?

Reply:
clangdev seems to be needed on Linux, otherwise stddef.h is not found

Reply:
stddef.h is not unique to clang it shouldn't require clangdev to be installed. Do you have a link to the spefic log output where this error occurred?

Reply:
Yes, you can find the log [here](https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=539883&view=logs&j=6f142865-96c3-535c-b7ea-873d86b887bd&t=22b0682d-ab9e-55d7-9c79-49f3c3ba4823&l=3980)

Reply:
@carterbox Is there anything I can do for this PR to be accepted? Another package that I maintain (qtmolecularnetwork) also use clang and clangdev and was accepted